docs/modules/ROOT/pages/tests-configuration.adoc

```diff
@@ -677,14 +677,33 @@ CAUTION: Use this tool at your own risk, since there is a chance of potential le
 
 === How to use an encrypted string in VIVIDUS
 
-. Add the secret key/text to the project properties as the value of:
+. Set the password in one of the following ways:
++
+.. Pass the key with its password-value into a test run using command line:
++
+[source,properties]
+----
+./gradlew runStories -Pvividus.encryptor.password=MYPAS_WORD
+----
++
+.. Set via OS environment variable `VIVIDUS_ENCRYPTOR_PASSWORD`:
++
+[source,properties]
+----
+export VIVIDUS_ENCRYPTOR_PASSWORD=MYPAS_WORD
+----
++
+.. Add the password to the project properties as the value of:
 +
 [source,properties]
 ----
 system.vividus.encryptor.password=MYPAS_WORD
 ----
 +
-IMPORTANT: This value must be kept secret and must not be committed to version control system.
+Keep in mind that a password specified in the command line takes priority over the environment variable,
+and the environment variable takes priority over the project property.
```

Comment:
I'm doubtful we allow to set password in a regular properties, afiak only system properties (we can declare them in files with `system.` prefix, but they are not regular) are allowed